enhancements/network/user-defined-network-segmentation.md

```diff
@@ -357,11 +359,106 @@ block/allow cluster egress or ingress traffic will still be enforced.
+| Role           | Select the network role in the pod, either `primary` or `secondary`.                                                                                                                                                                                                                                                                                                                               | No       |
+| MTU            | The maximum transmission unit (MTU).<br/>The default is 1400.                                                                                                                                                                                                                                                          | Yes      |
+| Subnets        | The subnet to use for the network across the cluster.<br/>E.g. 10.100.200.0/24.<br/>IPv6 (2001:DBB::/64) and dual-stack (192.168.100.0/24,2001:DBB::/64) subnets are supported.<br/>When omitted, the logical switch implementing the network only provides layer 2 communication, and users must configure IP addresses.<br/>Port security only prevents MAC spoofing if the subnets are omitted. | Yes      |
+| ExcludeSubnets | List of CIDRs and IP addresses.<br/>IP addresses are removed from the assignable IP address pool and are never passed to the pods.                                                                                                                                                                                                                                                                 | Yes      |
+
+The cluster scoped CRD should have the following additional field:
+
+| Field name | Description                                                                                           | optional |
+|------------|-------------------------------------------------------------------------------------------------------|----------|
+| Selector   | Key-value list of labels used as a selector for which namespace the network should be available for.  | No       |
+
+Since `localnet` topology for primary networks is non-goal, the CRDs controller should reject such requests.
+
+##### status
+The CRD status should reflect the NAD state through conditions.
+For example, when the NAD its created the condition should be placed with status `True`.
+
+For cluster scoped networks, the condition should be true once all desired namespaces are provisioned with the corresponding NAD.
+
+
+
+Granting permissions for users to create OVN networks in namespaces they are allowed should be manged using the RBAC mechanism.
+
+##### Example - Namespace scoped network
```

Comment:
we should not do namespace scoped network i think..

Reply:
Please see this thread https://github.com/openshift/enhancements/pull/1638/files#r1657307346